cmd/quadlet/main.go

```diff
@@ -103,28 +103,58 @@ func Debugf(format string, a ...interface{}) {
 func getUnitDirs(rootless bool) []string {
 	// Allow overriding source dir, this is mainly for the CI tests
 	unitDirsEnv := os.Getenv("QUADLET_UNIT_DIRS")
+	dirs := make([]string, 0)
+
 	if len(unitDirsEnv) > 0 {
-		return strings.Split(unitDirsEnv, ":")
+		if !isValidFilePath(unitDirsEnv) {
```

Comment:
`unitDirsEnv` is a colon delimited list of directories. So, once the list is longer than 1, `isValidFilePath` will return `false`, no?